docs/articles/index.html

```diff
@@ -0,0 +1,138 @@
+                <p>All you need is to have <a href="https://nodejs.org">Node.js</a> with <a href="https://www.npmjs.com">npm</a> installed and call a single command.</p>
+                <div class="command">npm install --save-dev testcafe</div>
+                <p>No browsers plugins to install, no binary compilation post-install steps.</p>
+            </div>
+        </div>
+        <div class="feature">
+            <span class="name">
+                Complete Test Harness
+            </span>
+            <div class="description">
+                <p>TestCafe provides all testing features: starting browsers, running tests and gathering results. You configure test run and start execution either via the API or from a command shell simply by running a single command</p>
+                <div class="command">testcafe safari,ie tests/</div>
+                <p>TestCafe aggregates test results from different browsers and outputs them into one comprehensive report.</p>
+            </div>
+        </div>
+        <div class="feature">
+            <span class="name">
+                Write Test Code Using ES2016
+            </span>
+            <div class="description">
+                <p>TestCafe tests are written with ES2016 using all latest JavaScript features like 'async/await'. TestCafe will compile your test code on-flight and run it immediately. It also ships with built-in support for source maps to leverage debugging experience. Source maps are automatically enabled, so all you need to do is start a debugging session in an IDE that supports them.</p>
+                <p>TestCafe introduces simple, but powerful test API. It offers a couple of dozen methods covering all user actions. Chained syntax produces code that is easy to write and read. Furthermore, you are free to use any assertion library to perform verifications of different kinds.</p>
+                <pre><code>import { expect } from 'chai';
+
+fixture `My Fixture`
+    .page('http://devexpress.github.io/testcafe/example');
+
+test('My Test', async t => {
+await t
+    .click('#send-button')
+    .handleAlert()
+    .typeText('#input', 'Peter Parker')
+    .wait(1000);
+
+expect(await t.eval(() => getSomethingOnTheClient())).to.be.true;
+});</code></pre>
+            </div>
+        </div>
+        <div class="feature">
+            <span class="name">
+                Write Regular Client JavaScript to Observe Page State
+            </span>
+            <div class="description">
+                <p>TestCafe introduces special kinds of functions that power you to observe the page state as if you are inside the page - `Selector` and `ClientFunction`. Simply write JavaScript code within these functions, it will be executed on the client side, giving you direct access to DOM elements or obtaining some data from the client. These functions are called from a test as a regular async function, so you can pass parameters to it and get the function result.</p>
```

Comment:
Please, revise the whole composition of this paragraph. Don't start with `special kind of functions`. It's lame.

Start with
`...observe the page state...`
`...code executed on the client side...`
`...direct access to DOM elements or obtaining some data from the client...`

Saying by which means you can do this is optional in this marketing part. At least, it shouldn't go at the first place.